west.yml

```diff
@@ -126,6 +126,10 @@ manifest:
     - name: cmsis-nn
       revision: ea987c1ca661be723de83bd159aed815d6cbd430
       path: modules/lib/cmsis-nn
+    - name: cmsis_6
```

Comment:
do we really want to have both CMSIS 5 and CMSIS 6 in the manifest ?
and if yes, why ?

Reply:
https://github.com/zephyrproject-rtos/zephyr/issues/77220#issuecomment-2331879392

Reply:
Thanks, that piece of info had been nice to have in the PR description.

you can still add it, so that others easily find it and not have to scroll through all comments :wink:

Reply:
Yeah true. I had already linked the issue from this PR but it's not that visible without the text in the PR description.